docs/proposals/work-pause/README.md

```diff
@@ -0,0 +1,625 @@
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: nginx-propagation
+spec:
+  resourceSelectors:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: nginx
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+        - member3
+  pauseStrategy:
+    distributionPauseStrategy:
+      pauseClusterAffinity:
+        clusterNames:
+          - member2
+          - member3
+```
+
+<!--
+当用户确定 member1 中的 deployment 资源发布完成时，修改同步配置如下时，即可继续将新的资源版本同步至 member2 集群中：
+-->
+
+When the user confirms that the deployment resource in member1 has been successfully published, and modifies the synchronization configuration as follows, the new version of the resource can then be synchronized to the member2 cluster:
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: nginx-propagation
+spec:
+  resourceSelectors:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: nginx
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+        - member3
+  pauseStrategy:
+    distributionPauseStrategy:
+      pauseClusterAffinity:
+        clusterNames:
+          - member3
+```
+
+<!--
+如果在将新的资源版本同步至 member1 成员集群后，发现该应用出现问题，导致发布失败，用户可以在 Karmada 控制面中将 Deployment 资源版本回退，从而保护 member2、member3集群中的业务不受影响。
+-->
+
+If, after synchronizing the new version of the resource to the member1 cluster, issues are found with the application that cause the release to fail, the user can roll back the version of the Deployment resource in the Karmada control plane, thereby protecting the business in the member2 and member3 clusters from being affected.
+
+Users set the `Deployment(default/nginx)` to be paused for scheduling in the `PropagationPolicy`, and as a result, the resource will not be scheduled or distributed.
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: nginx-propagation
+spec:
+  resourceSelectors:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: nginx
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+        - member3
+  pauseStrategy:
+    schedulePause: true
+```
+
+And then, the related ResourceBinding will be in a paused state.
+
+```yaml
+apiVersion: work.karmada.io/v1alpha2
+kind: ResourceBinding
+metadata:
+  name: nginx-deployment
+  namespace: default
+spec:
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+        - member3
+  resource:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: nginx
+    namespace: default
+status:
+  conditions:
+  - lastTransitionTime: "2024-07-01T08:33:28Z"
+    message: Scheduling is in a paused state.
+    reason: SchedulePause
+    status: "True"
+    type: SchedulePause
```

Comment:
Who will update this and when wil this condition be updated? Will this be remove when rb.PauseStrategy.SchedulePause is set to false?

Reply:
Good question!
I think the binding-status-controller will take care of this task.

> Will this be remove when rb.PauseStrategy.SchedulePause is set to false?

The status will be set to false when rb.PauseStrategy.SchedulePause is set to false.